.golangci.yaml

```diff
@@ -8,28 +8,37 @@ run:
 linters:
   disable-all: true
   enable:
+  - asasalint
   - asciicheck
+  - bidichk
   - bodyclose
+  #- containedctx
   - dogsled
-  # - errcheck
-  # - errorlint
+  - dupword
+  - durationcheck
+  - errcheck
+  - errchkjson
   - exportloopref
   - gci
+  - ginkgolinter
   - goconst
   - gocritic
   - godot
   - gofmt
   - goimports
+  - goprintffuncname
+  - gosec
   - gosimple
   - govet
   - importas
-  - gosec
   - ineffassign
+  - loggercheck
   - misspell
-  # - nakedret
-  # - nilerr
-  # - noctx
+  - nakedret
```

Comment:
I'm not convinced that this is an improvement, but I respect that other people seem to have a different opinion.